<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serial_port</class>
 <widget class="QMainWindow" name="serial_port">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>659</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>803</width>
    <height>659</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>803</width>
    <height>659</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Qt信号调试处理程序</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>133</width>
      <height>166</height>
     </rect>
    </property>
    <property name="title">
     <string>基本设置</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>串口</string>
         </property>
         <property name="buddy">
          <cstring>cbPortName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbPortName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>波特率</string>
         </property>
         <property name="buddy">
          <cstring>cbBaudRate</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbBaudRate"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>数据位</string>
         </property>
         <property name="buddy">
          <cstring>cbdata</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbdata"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>校验位</string>
         </property>
         <property name="buddy">
          <cstring>cbPairty</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbPairty"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>停止位</string>
         </property>
         <property name="buddy">
          <cstring>cbStop</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbStop"/>
       </item>
      </layout>
     </item>
    </layout>
    <zorder></zorder>
    <zorder></zorder>
    <zorder></zorder>
    <zorder></zorder>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>791</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>641</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="hexLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>16进制显示</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>582</y>
      <width>522</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>com口状态：</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbstate">
       <property name="text">
        <string>com1已关闭</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>138</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>发送：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>88</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>接收：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>641</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="editData">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>641</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="ascdata">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>661</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="outRangeEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>661</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>121</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="saveLocal">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveAdditon">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>另存为</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendData">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="clearWindow">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>清空显示</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearFile">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendFile">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>发送文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>450</y>
      <width>20</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>发送设置</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>120</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="hexSend">
        <property name="text">
         <string>16进制发送</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showAscii">
        <property name="text">
         <string>显示ASCII码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enterSend">
        <property name="text">
         <string>加回车发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="autosend">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>自动发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearsend">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>清除发送 </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>191</y>
      <width>77</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="openbutton">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>open</string>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopShow">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>631</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="retbt">
    <property name="geometry">
     <rect>
      <x>704</x>
      <y>580</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>groupBox</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>hexLabel</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>line</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>textBrowser</zorder>
   <zorder>retbt</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQt">
    <property name="title">
     <string>Base</string>
    </property>
   </widget>
   <addaction name="menuQt"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="actionFFT_transform">
   <property name="text">
    <string>FFT transform</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
